orderer/consensus/etcdraft/node.go

```diff
@@ -201,6 +202,14 @@ func (n *node) send(msgs []raftpb.Message) {
 
 		status := raft.SnapshotFinish
 
+		// Replace node list in snapshot with CURRENT node list in cluster.
```

Comment:
can you explain why you're doing this?

Reply:
this is because of changes in the way raft nodes are restarted with this new version. There is a change in the API of etcd raft that requires that we restart without setting peers and set them afterwards.

I fished that change from another in-progress PR that never made it to main, with better explanations: https://github.com/hyperledger/fabric/pull/2452#issuecomment-792312984